ibis/pyspark/tests/test_window.py

```diff
@@ -0,0 +1,51 @@
+import pandas.util.testing as tm
+import pyspark.sql.functions as F
+import pytest
+from pyspark.sql.window import Window
+
+import ibis
+
+pytest.importorskip('pyspark')
+pytestmark = pytest.mark.pyspark
+
+
+def test_time_indexed_window(client):
```

Comment:
Can you parameterize this test by the window bounds?

```
(trailing_window(ibis.interval(hours=1), ...), Window.partitionBy(...).rangeBetween(...))

(ibis.range_window(..., ...), 
Window.partitionBy(...).rangeBetween(...))
```

Reply:
We cannot simply put Window in param because that might trigger error of `'NoneType' object has no attribute '_jvm'` where spark jvm is not initialized properly.  Instead, I put range as param.